pyposeidon/boundary.py

```diff
@@ -59,6 +59,12 @@ def __init__(self, geometry, coastlines=None, cbuffer=None, blevels=None, **kwar
             dem_source (str): Path or url to bathymetric data.
         """
 
+        geometry = kwargs.get("geometry", None)
+        coastlines = kwargs.get("coastlines", None)
+        cbuffer = kwargs.get("cbuffer", None)
+        blevels = kwargs.get("blevels", None)
+        prad = kwargs.get("R", 1.0)
+
```

Comment:
The `kwargs` are not needed here. Rewriting the `__init__()` definition as:

``` python
def __init__(self, geometry=None, coastlines=None, cbuffer=None, blevels=None, R=1.0):
    ...
```
among other things, is going to make the function more readable, shorter, less error prone to typos when you call the function, the API documentation will just work etc etc